src/NuGet.Core/NuGet.Build.Tasks/GetRestoreSolutionProjectsTask.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
```

Comment:
I think we need to decide whether header is needed or not and be consistent about it.